examples/ai-ml/ray/sources/hface_t5_summarize_serve.py

```diff
@@ -0,0 +1,23 @@
+# source: https://github.com/ray-project/ray/blob/master/python/ray/serve/examples/doc/e2e_class_deployment.py
+from importlib_metadata import version
+import ray
+from ray import serve
+from transformers import pipeline
+
+
+ray.init(address="ray://raycluster-autoscaler-head-svc:10001", namespace="serve")
```

Comment:
is this port correct or should it be `8000` based on [this](https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/871/files#diff-246baec33444449358c413d7033d77d25164b39fb5605e6212e8da1821750bd9R26) ?

Reply:
nope its the right port for job submission